UI/window-basic-properties.cpp

```diff
@@ -408,11 +408,11 @@ void OBSBasicProperties::DrawTransitionPreview(void *data, uint32_t cx,
 {
 	OBSBasicProperties *window = static_cast<OBSBasicProperties *>(data);
 
-	if (!window->source)
+	if (!window->sourceClone)
 		return;
 
-	uint32_t sourceCX = max(obs_source_get_width(window->source), 1u);
-	uint32_t sourceCY = max(obs_source_get_height(window->source), 1u);
+	uint32_t sourceCX = max(obs_source_get_width(window->sourceClone), 1u);
+	uint32_t sourceCY = max(obs_source_get_height(window->sourceClone), 1u);
```

Comment:
What's the story behind this?

Reply:
To preview transitions the properties window creates a clone of the transition with a source A and B to transition between. It was using the size of original transition which was not always set to display the clone which has a fixed size. So without this change the A and B where not always in the center of the preview.